.github/workflows/publish.yml

```diff
@@ -38,20 +39,20 @@ jobs:
           password: ${{ secrets.GITHUB_TOKEN }}
       - name: Prepare environment variables
         run: |
-          if [[ "${{ matrix.network }}" == "testnet" ]]; then
+          if [[ "${{ matrix.network }}" == "zen" ]]; then
             echo "BUILD_TAGS=testnet netgo" >> $GITHUB_ENV
-            echo "DOCKER_METADATA_suffix=-testnet" >> $GITHUB_ENV
+            echo "DOCKER_METADATA_SUFFIX=-zen" >> $GITHUB_ENV
```

Comment:
This is inconsistent with `hostd` but I think it's more correct, especially since we're adding a v2 testnet soon this might be necessary to update in `hostd`? Not sure but decided to suggest it.